IrConsoleApp/Program.cs

```diff
@@ -3,15 +3,149 @@
 
 namespace IrynaZharnasek_SaysHello
 {
-    class Program { 
-        static void Main() {
-            Console.WriteLine("Hi! What is your name?");
-
-            string st1;
-            
-            st1 = (Console.ReadLine());
-            Console.WriteLine("Nice to meet you " + st1);
-        } 
+    class Program {
+        static void Main()
+        {
+            Task1();
+            Task2();
+            Task3();
+            Task4();
+            Task5();
+            Task6();
+            Task7();
+            Task8();
+            Task9();
+        }
+      
+    private static void Task1()
+    {
+        var num = 20 + 5;
+        Console.WriteLine(num);
+    }
+
+    private static void Task2()
+    {
+        Console.WriteLine("Enter count of days from 1000 to 9999:");
+        ushort Days = (ushort)Convert.ToUInt32(Console.ReadLine());
```

Comment:
Please use camel case for Days variable (shoud be "days"). Super useful link with coding guidlines - https://wiki.itechart-group.com/display/codstandarts/Coding+Standard+Visual+C+Sharp

Reply:
Done.